{
    "original_source": {
        "database(url)": "https://www.cve.org",
        "software_system(CPE)": [
            "cpe:2.3:a:lfprojects:case_python_utilities:0.5.0:*:*:*:*:python:*:*"
        ],
        "fix_commit": "https://github.com/Cyber-Domain-Ontology/CDO-Utility-Local-UUID/commit/9e78f7cb1075728d0aafc918514f32a1392cd235",
        "version_end_excluding": {
            "lfprojects:case_python_utilities": "unknown"
        },
        "path": {
            "file_path": ""
        }
    },
    "programming_language": "PYTHON",
    "licences": {
        "database": "",
        "affected_repository": "Apache License 2.0"
    },
    "related_vulnerability_entry": {
        "CVE": [
            "CVE-2024-22194"
        ],
        "CWE": [
            "CWE-215",
            "CWE-337"
        ]
    }
}